tests/search.test.ts

```diff
@@ -134,6 +135,34 @@ describe.each([
     expect(response.hits.length).toEqual(2)
   })
 
+  test(`${permission} key: Basic phrase searchwith optionalWords at none`, async () => {
+    const client = await getClient(permission)
+    const response = await client
+      .index(index.uid)
+      .search('"french book" about', { optionalWords: OptionalWords.NONE })
+
+    expect(response).toHaveProperty('hits', expect.any(Array))
+    expect(response).toHaveProperty('offset', 0)
+    expect(response).toHaveProperty('limit', 20)
+    expect(response).toHaveProperty('processingTimeMs', expect.any(Number))
```

Comment:
Do you really need to assert this 😬?

Reply:
I can say that for other assertions like `hits`, `offset` and `limit`.

You can create a custom matcher or use one if it exists that could assert that all in once pseudocode:

```js
expect(response).toBeValidSearchResponse() 
```

inside of this matcher, you can assert those properties, because I assume you are not interested in the values just in the properties name.

Reply:
Thats a nice suggestion! But you are right about these assertions not being useful. I removed them :)